integrations/cdi/datasource-hikaricp/src/main/java/io/helidon/integrations/datasource/hikaricp/cdi/config/package-info.java

```diff
@@ -15,9 +15,17 @@
  */
 
 /**
- * Provides classes and interfaces marrying MicroProfile Config
- * constructs and {@link
+ * Provides classes and interfaces marrying <a
+ * href="https://microprofile.io/project/eclipse/microprofile-config"
+ * target="_parent">MicroProfile Config</a> constructs and {@link
  * io.helidon.service.configuration.api.ServiceConfiguration}
- * constructs for the Hikari Connection Pool.
+ * constructs for the <a
+ * href="https://github.com/brettwooldridge/HikariCP/blob/HikariCP-2.7.8/README.md#-hikaricpits-fasterhikari-hikal%C4%93-origin-japanese-light-ray"
+ * target="_parent">Hikari Connection Pool</a>.
+ *
+ * @author <a href="mailto:laird.nelson@oracle.com">Laird Nelson</a>
```

Comment:
Please remove `@author` annotation from your template ;). Also once you rebase on latest master, checkstyle will fail on these lines.